ttnn/cpp/ttnn/tensor/types.hpp

```diff
@@ -667,10 +667,12 @@ struct Shape {
     // It is used to flip the default value of operator[] to return the shape without padding
     tt::tt_metal::LegacyShape value;
 
-    explicit Shape(const tt::tt_metal::LegacyShape &shape) : value{shape} {}
+    Shape(const std::initializer_list<uint32_t> dimensions) : value{dimensions} {}
```

Comment:
Do we have a move constructor?'It would be nice to add it.

Reply:
We have a default one in `LegacyShape`. I think this can be done later.